src/frontend/apps/impress/src/features/docs/doc-tree/components/DocSubPageItem.tsx

```diff
@@ -116,9 +116,11 @@ export const DocSubPageItem = (props: TreeViewNodeProps<Doc>) => {
           ? 'var(--c--theme--colors--greyscale-100)'
           : 'var(--c--theme--colors--greyscale-000)'};
         .light-doc-item-actions {
-          display: ${menuOpen || !isDesktop ? 'flex' : 'none'};
-          position: absolute;
+          display: flex;
+          position: sticky;
           right: 0;
+
+          opacity: ${menuOpen || !isDesktop ? '1' : '0'};
```

Comment:
By doing like that the name is cropped far before needed, because the actions button take place even when not displayed.